napari/layers/base/base.py

```diff
@@ -139,6 +139,7 @@ def __init__(
         self._position = (0,) * self.dims.ndisplay
         self.is_pyramid = False
         self._editable = True
+        self._editable_history = True
```

Comment:
what's this for?

Reply:
a mistake, was playing around with it in an earlier version of PR, will cut

Reply:
cut